packages/web/src/shared/components/form/AutocompleteInput.tsx

```diff
@@ -112,9 +134,9 @@ const AutocompleteInput = forwardRef<HTMLInputElement, AutocompleteInputProps>(
                   case 'Enter':
                   case 'Tab': {
                     if (typeof activeIndex === 'number') {
-                      change(filteredSuggestions[activeIndex]);
+                      change(filteredSuggestions[activeIndex], e.key === 'Tab');
```

Comment:
```suggestion
                      change(filteredSuggestions[activeIndex], true);
```

Reply:
This condition is on purpose because we don't want the Tab key to signify approval since it used for navigation between interactive elements. In that sense, Tab is "implicit" because it is the keyboard version of clicking out of the input without making a selection

Reply:
Yeah I understood that. This is inside a `case 'Tab':` block, so the expression `e.key === 'Tab'` will always evaluate to `true`. What am I missing